third_party/xls_go_math/fpexp_32.x

```diff
@@ -0,0 +1,244 @@
+// Copyright 2021 The XLS Authors
```

Comment:
This should be licensed like this file:

https://github.com/google/xls/blob/main/third_party/xls_go_math/fpsqrt_32.x

No "Licensed under the Apache License...". Same with other files.

Reply:
Done.
fpexp_32.x is the only file in this PR I can see where this change needs to be made - am I missing others?